test/react-web/client/graphql/queries.test.tsx

```diff
@@ -514,6 +514,61 @@ describe('queries', () => {
     renderer.create(<ApolloProvider client={client}><Parent /></ApolloProvider>);
   });
 
+  it('allows you to skip then unskip a query with new options (top-level syntax)', (done) => {
+    const query = gql`query people($first: Int) { allPeople(first: $first) { people { name } } }`;
+    const dataOne = { allPeople: { people: [ { name: 'Luke Skywalker' } ] } };
+    const dataTwo = { allPeople: { people: [ { name: 'Leia Skywalker' } ] } };
+    const networkInterface = mockNetworkInterface(
+      { request: { query, variables: { first: 1 } }, result: { data: dataOne } },
+      { request: { query, variables: { first: 2 } }, result: { data: dataTwo } },
+      { request: { query, variables: { first: 2 } }, result: { data: dataTwo } },
```

Comment:
> my mocking of the network interface may be to blame

I doubt it! (Although we should find a way to use the MNI from AC.

> so I added the extra test

Oh, what test are you referring to? The above is my own test that I just wrote.